src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs

```diff
@@ -2248,6 +2334,26 @@ public StringBuilder Replace(char oldChar, char newChar, int startIndex, int cou
             return this;
         }
 
+        public StringBuilder Replace(Rune oldRune, Rune newRune)
+        {
+            return Replace(oldRune, newRune, 0, Length);
+        }
+        public StringBuilder Replace(Rune oldRune, Rune newRune, int startIndex, int count)
+        {
+            // Convert oldRune to span
+            Span<char> leftChars = stackalloc char[2];
```

Comment:
You don't have to do this. just will be nice to name the variables with `old` and `new` instead of `left` and `right`.